src/rtext.c

```diff
@@ -1421,6 +1421,26 @@ int TextToInteger(const char *text)
     return value*sign;
 }
 
+float TextToFloat(const char *text) {
+    float value = 0;
+    float sign = 1;
+
+    if ((text[0] == '+') || (text[0] == '-')) {
+        if (text[0] == '-') sign = -1;
+        text++;
+    }
+    int i;
+    for (i = 0; ((text[i] >= '0') && (text[i] <= '9')); ++i) value = value * 10 + (float) (text[i] - '0');
+    if (text[i++] == '.') {
+        float divisor = 10;
+        for (; ((text[i] >= '0') && (text[i] <= '9')); ++i) {
+            value += (float) (text[i] - '0') / divisor;
+            divisor *= 10;
+        }
+    }
+    return value * sign;
```

Comment:
There is also a function TextToInteger which is used in the GuiValueBox control, and since i needed a Valuebox that supports float values i thought i would adapt the function to convert also to float.
The same question could be asked for TextToInteger and atoi

Reply:
About the addition of that function, it was for library consistency in functions naming and also when exposed through other language bindings that do not have direct access to C library.

About the use of a custom implementation instead of the standard library provided one (that probably is much better), there was a time in raylib development that I tried to minimize dependencies on C standard library and directly implement some of its functions, also useful for students checking the code.

I think the function can be added with some notes and maybe enable a flag to choose the desired implementation. I can review that part myself more carefully.